### PR TITLE
Add 'https://www.patterns.dev/' to the resources page (https://github…

### PR DESCRIPTION
**Pull request by lazarnarcis**
….com/FrontEnd-ro/frontend.ro/issues/486)

### CODE REVIEW

File: client/components/learn/Resources/resources-model.ts [hunk 1]
```diff
@@ -41,6 +41,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://validator.w3.org/',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/w3c-validator_500w.png`,
     },
+    {
+      title: 'Patterns Dev',
```
Comment (iampava) [inline client/components/learn/Resources/resources-model.ts:45]: Please move in the books section. Put it in second place, after "You Don't Know JS"

File: client/components/learn/Resources/resources-model.ts [hunk 2]
```diff
@@ -41,6 +41,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
...
+      url: 'https://patterns.dev/',
+      cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/patterns.dev.png`,
```
Comment (iampava) [inline client/components/learn/Resources/resources-model.ts:47]: Please resize the asset to `500px` width. Then rename to `patters-dev_500w.png` (notice the `_500w` at the end, it's the same as all other assets).

**Review by iampava — CHANGES_REQUESTED** [review]
Overall looks good. A few changes only

**Review by lazarnarcis — COMMENTED** [review]
i think everything is ok now

File: client/components/learn/Resources/resources-model.ts [hunk 3]
```diff
@@ -232,6 +232,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://github.com/getify/You-Dont-Know-JS',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/ydkjs_500w.png`,
     },
+    {
+      title: 'Patterns Dev',
+      url: 'https://patterns.dev/',
+      cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/patterns.dev.png`,
```
Comment (iampava) [inline client/components/learn/Resources/resources-model.ts:238]: Here you have `patterns.dev.png` but the actual file name should be `patterns-dev_500w.png` (please rename the png file. You have a typo. Should be **patterns-dev_500w.png**.)

**Review by iampava — CHANGES_REQUESTED** [review]
Almost there

**Review by lazarnarcis — COMMENTED** [review]
now is ok :)

**Review by iampava — APPROVED** [review]
Great!

**Pull request merged**